cl/corpus_importer/management/commands/update_casenames_wl_dataset.py

```diff
@@ -126,6 +126,88 @@ def make_citations(cite1, cite2) -> list[FullCaseCitation]:
     return cites
 
 
+def find_matches(
+    valid_citations: list[FullCaseCitation],
+    csv_docket_num: str,
+    csv_date_filed: date,
+    csv_case_name: str,
+) -> list[tuple[OpinionCluster, str]]:
+    """Find matches for row data
+
+    :param valid_citations: list of FullCaseCitation objects
+    :param csv_docket_num: cleaned docket number from row
+    :param csv_date_filed: formatted filed date from row
+    :param csv_case_name: case name from csv row
+    :return: list of tuples of matched OpinionCluster and used citation
+    """
+    matches = []
+
+    # Try to match row using both citations
+    for citation in valid_citations:
+
+        possible_matches = Citation.objects.filter(
+            **citation.groups,
+            cluster__docket__docket_number__contains=csv_docket_num,
+            cluster__date_filed=csv_date_filed,
+        )
+        if not possible_matches:
+            # Match not found with citation, docket number and date filed
```

Comment:
Log?

Reply:
The log is after the find_matches function, wait for it to finish checking the two citations.

If there are no matches with the two citations then log the message.